setup.cfg

```diff
@@ -50,9 +50,21 @@ tests =
     flexmock>=0.10.4
     pytest>=5.2.4
     pytest-cov
+    pytest-black
 
 [options.packages.find]
 exclude = tests
 
 [tool:pytest]
-addopts = --cov=apigentools --cov-config .coveragerc --cov-report=term-missing
+addopts = --black --cov=apigentools --cov-config .coveragerc --cov-report=term-missing
+
+[flake8]
+max-line-length = 88
+extend-ignore = E203
+
+[tool:isort]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
```

Comment:
line length of 120 :pretty_please:

Reply:
I am usually for 78, but I keep the default of black.

Reply:
This is only to make sure that `isort` and `black` have the same setting.

Reply:
My personal view is that 120 char for Python is too much (and for Java too little 😉).

Reply:
I usually prefer to go for 100 (or 99), as I find it to be a reasonable number of characters to have 2 terminals open side-by-side without line wrapping (with a reasonable font size on a resonably-sized display ;)) PEP 8 suggests that it's ok to keep line length 99 at max [1], so that's what I'd prefer.

[1] https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
We've aligned with other Datadog repositories on 120.